MapboxCoreNavigationTests/CLLocationTests.swift

```diff
@@ -11,42 +11,74 @@ class CLLocationTests: XCTestCase {
         let horizontalAccuracy: CLLocationAccuracy = 50
+        let verticalAccuracy: CLLocationAccuracy = 50
+        let speedAccuracy: CLLocationAccuracy = 25
+        let bearingAccuracy: CLLocationAccuracy = 35
         
-        let fixLocation = FixLocation(coordinate: coordinate, time: timestamp,
-                                        speed: speed as NSNumber, bearing: bearing as NSNumber,
-                                        altitude: altitude as NSNumber, accuracyHorizontal: horizontalAccuracy as NSNumber, provider: nil)
+        let fixLocation = FixLocation(coordinate: coordinate,
+                                      time: timestamp,
+                                      speed: speed as NSNumber,
+                                      bearing: bearing as NSNumber,
+                                      altitude: altitude as NSNumber,
+                                      accuracyHorizontal: horizontalAccuracy as NSNumber,
+                                      provider: nil,
+                                      bearingAccuracy: bearingAccuracy as NSNumber,
+                                      speedAccuracy: speedAccuracy as NSNumber,
+                                      verticalAccuracy: verticalAccuracy as NSNumber)
         
         let location = CLLocation(fixLocation)
         
+        XCTAssertEqual(location.coordinate.latitude, coordinate.latitude)
+        XCTAssertEqual(location.coordinate.longitude, coordinate.longitude)
         XCTAssertEqual(location.coordinate, fixLocation.coordinate)
         XCTAssertEqual(location.timestamp, fixLocation.time)
         XCTAssertEqual(location.speed, fixLocation.speed?.doubleValue)
         XCTAssertEqual(location.altitude, fixLocation.altitude?.doubleValue)
         XCTAssertEqual(location.horizontalAccuracy, fixLocation.accuracyHorizontal?.doubleValue)
+        XCTAssertEqual(location.speedAccuracy, fixLocation.speedAccuracy?.doubleValue)
+        XCTAssertEqual(location.verticalAccuracy, fixLocation.verticalAccuracy?.doubleValue)
+        XCTAssertEqual(location.course, fixLocation.bearing?.doubleValue)
+        if #available(iOS 13.4, *) {
+            XCTAssertEqual(location.courseAccuracy, fixLocation.bearingAccuracy?.doubleValue)
+        }
     }
 
     func testShiftLocation() {
         let coordinate = CLLocationCoordinate2D(latitude: 1, longitude: 2)
-        let location = CLLocation(coordinate: coordinate, altitude: 10,
+        let location = CLLocation(coordinate: coordinate,
+                                  altitude: 10,
                                   horizontalAccuracy: 40,
                                   verticalAccuracy: 50,
-                                  course: 180, speed: 18,
+                                  course: 180,
+                                  speed: 18,
                                   timestamp: Date())
         
         XCTAssertEqual(location.timestamp, location.shifted(to: location.timestamp).timestamp)
```

Comment:
Does that make sense to use different timestamp for the test? Otherwise looks like the test does not do the job.

Reply:
Makes sense, updated.